include/client.h

```diff
@@ -86,6 +87,7 @@ struct Attributes {
     // It is not easy for union to support them.
     std::string str_v;  // for both STRING and BYTES
     std::chrono::time_point<std::chrono::system_clock> time_v;
+    ::google::protobuf::Duration duration;
```

Comment:
Shouldn't this be std::chrono::duration, to be consistent with how time is handled? Alternatively, the time value could be changed to ::google::protobuf::Timestamp. Either way works, but I do think we should be consistent.